### PR TITLE
Map

### PR DESCRIPTION
**Pull request by AdamBeresnev**

### CODE REVIEW

File: WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/MapViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using ComparisonEngineUI.Views;
+using Xamarin.Forms;
```
Comment (RaimondSob) [inline WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/MapViewModel.cs:5]: There should be an empty space between using and namespace

File: WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/AboutViewModel.cs [hunk 2]
```diff
@@ -2,14 +2,21 @@
...
 using Xamarin.Essentials;
 using Xamarin.Forms;
+using ComparisonEngineUI.Views;
 
 namespace ComparisonEngineUI.ViewModels
 {
     public class AboutViewModel : BaseViewModel
     {
+        public Command OpenMap { get; }
         public AboutViewModel()
```
Comment (ElektrosStulpas) [inline WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/AboutViewModel.cs:12]: should be whitespace between these two lines

File: WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/MapViewModel.cs (hunk 1 above)
Comment (AdamBeresnev) [reply WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/MapViewModel.cs:5]: Fixed

**Review by ElektrosStulpas — COMMENTED** [review]
hard to comment, but didn't find anything worrying

**Pull request merged**